packages/shared/src/components/post/collection/CollectionSubscribeButton.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import {
+  checkHasStatusPreference,
+  useNotificationPreference,
+} from '../../../hooks/notifications';
+import { NotificationType } from '../../notifications/utils';
+import { NotificationPreferenceStatus } from '../../../graphql/notifications';
+import BellDisabledIcon from '../../icons/Bell/Disabled';
+import { Post } from '../../../graphql/posts';
+import { Button } from '../../buttons/Button';
+import BellIcon from '../../icons/Bell';
+import { SimpleTooltip } from '../../tooltips';
+
+export type CollectionSubscribeButtonProps = {
+  post: Post;
+  isCondensed?: boolean;
+};
+
+type CollectionSubscribeButtonViewProps = {
+  isSubscribed: boolean;
+  isFetching: boolean;
+  onClick: () => void;
+};
+
+const CollectionSubscribeButtonRegular = ({
+  isSubscribed,
+  isFetching,
+  onClick,
+}: CollectionSubscribeButtonViewProps): ReactElement => {
+  return (
+    <Button
+      className={classNames(
+        'btn',
+        isSubscribed ? 'btn-secondary' : 'btn-primary',
+      )}
+      icon={isSubscribed ? <BellDisabledIcon /> : <BellIcon />}
+      disabled={isFetching}
+      onClick={onClick}
+    >
+      {isSubscribed ? 'Unsubscribe' : 'Subscribe'}
+    </Button>
+  );
+};
+
+const CollectionSubscribeButtonCondensed = ({
+  isSubscribed,
+  isFetching,
+  onClick,
+}: CollectionSubscribeButtonViewProps): ReactElement => {
+  return (
+    <SimpleTooltip content={isSubscribed ? 'Unsubscribe' : 'Subscribe'}>
+      <Button
+        className={classNames('btn btn-tertiary')}
```

Comment:
Same goes here I believe.
```suggestion
        className="btn-tertiary"
```